_posts/2020-10-06-solidity-dynamic-array-cleanup-bug.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+published: true
+title:  "Solidity Dynamic Array Cleanup Bug"
+date:   2020-04-06
+author: Solidity Team
+category: Security Alerts
+---
+
+On Sept 17, 2020 a bug in the Solidity code generator was found. The bug is fixed with version 0.7.3
```

Comment:
Two different date formats. I think in this text, the first but with full month name would be best.